src/components/EventList/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import Table from '../Table';
+import styles from './styles';
+
+const columns = [
+  { title: 'Event Name', field: 'title' },
+  { title: 'start Time', field: 'startDateString' },
+  { title: 'end Time', field: 'endDateString' },
+];
+
+function EventList({ events, handleEventClick }) {
+  // show up an event Card which clicked by passing callback
+
+  const listEvents = [];
+  for (let i = 0; i < events.length; i += 1) {
+    const listEvent = {
+      id: events[i].id,
+      title: events[i].title,
+      startDateString: moment(events[i].startDate).format(
```

Comment:
It looks like listEvents just needs to render the columns for EventName, StartTimeString, and EndTimeString, so I think it would be better if we just store those fields in the listEvent objects since the other fields are unused.

Reply:
In the eventDetails card the venue is needed to render, also the id is needed to identify which card the user is clicking on so that it can hide the card. Also I think id may be needed when the "see more" button is clicked on the event card. I can remove the startDate and endDate from listEvent

Reply:
sounds good - thanks for explaining :) My main gripe was with the duplication of startDate and startDateString